browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts

```diff
@@ -155,8 +151,6 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
       await validateScreenshot(
```

Comment:
**suggestion:** I think this screenshot pretty much has everything that the others do plus more (since it has the additional program and the extra buttons for a guest). I think we take a screenshot of this page (in the various sizes) and we remove the other screenshots, since we have expectCommonIntakeConfirmationPageNorthStar, which validates that it shows correctly for a TI / without programs / etc.